src/Compilers/Core/MSBuildTask/Microsoft.Managed.Core.targets

```diff
@@ -56,8 +56,12 @@
   <Target Name="DiscoverEditorConfigFiles" Condition="'$(DiscoverEditorConfigFiles)' != 'false'" BeforeTargets="CoreCompile">
     <DiscoverEditorConfigFiles InputFiles="@(Compile);@(AdditionalFile)">
       <Output TaskParameter="EditorConfigFiles" ItemName="EditorConfigFiles" />
+      <Output TaskParameter="PotentialEditorConfigFiles" ItemName="PotentialEditorConfigFiles" />
     </DiscoverEditorConfigFiles>
   </Target>
+  
+  <!-- Find and return the files paths where we looked for .editorconfig files but did not find one. -->
```

Comment:
Is this used for CPS, and the host object for csproj.dll/msvbprj.dll? Can we unify?

Reply:
It will be used by CPS and by the DTBB path in csproj/msvbprj, not by the host object. Once the host object goes away, it will be used everywhere.

The DTBB path is designed around collecting the compiler arguments to pass along to the compiler. In cases where it needs information that isn't passed to the compiler (like the potential analyzer config files) we need a separate target to pull it out. The `<Link>` metadata on `<Compile>` items is handled the same way, though in that case there was already a target that would provide the items I needed.